lib/sshkit/backends/local.rb

```diff
@@ -11,6 +11,7 @@ def initialize(_ = nil, &block)
       end
 
       def upload!(local, remote, options = {})
+        remote = File.join(pwd_path, remote) unless remote[0] == "/"
```

Comment:
Nitpick: I'd prefer `remote.start_with?("/")` instead of `remote[0] == "/"` here and elsewhere, for readability.

Reply:
Oh yeah that's much nicer. 👍

I always avoid `#start_with?` thinking it was ActiveSupport or introduce in Ruby 2+, but it's been [around since 1.8.7](https://ruby-doc.org/core-1.8.7/String.html#method-i-start_with-3F). 🙈